sdk/communication/communication-administration/review/communication-administration.api.md

```diff
@@ -459,25 +489,32 @@ export interface ReleaseResponse {
 // @public
 export type ReleaseStatus = "Pending" | "InProgress" | "Complete" | "Failed" | "Expired";
 
+// @public
+export type SearchOptions = OperationOptions;
+
 // @public
 export type SearchStatus = "Pending" | "InProgress" | "Reserved" | "Expired" | "Expiring" | "Completing" | "Refreshing" | "Success" | "Manual" | "Cancelled" | "Cancelling" | "Error" | "PurchasePending";
 
 // @public
 export type TokenScope = "chat" | "voip" | "pstn";
 
+// @public
+export type UnconfigurePhoneNumberOptions = OperationOptions;
```

Comment:
Sorry to make you revert this change; ConfigurePhoneNumber and UnconfigurePhoneNumber are no longer LROs (Long running operations).